Packs/GCP/Integrations/GCP/GCP.py

```diff
@@ -2031,6 +2038,335 @@ def get_credentials(args: dict, params: dict) -> Credentials:
+    """
+    Creates a subnet-mode network in the specified project using the data provided.
+    Args:
+        creds (Credentials): GCP credentials.
+        args (dict[str, Any]): Arguments including:
+            - name: Name of the network
+            - description: Optional description
+            - autoCreateSubnetworks: "true"/"false" (optional, defaults to True)
+            - routing_config_routing_mode: "REGIONAL" or "GLOBAL" (optional)
+            - project_id: GCP project ID
+    Returns:
+        CommandResults: The result of the network creation operation.
+    """
+    config = {}
+
+    # Name (required)
+    if args.get("name"):
+        name = args.get("name", "").lower()
+        config["name"] = name
+    else:
+        raise ValueError("The 'name' argument is required to create a network.")
+
+    # Description (optional)
+    if args.get("description"):
+        config["description"] = args.get("description")
+
+    # Auto-create subnets (default to True)
+    auto_create_sub_networks = args.get("autoCreateSubnetworks")
+    if auto_create_sub_networks is not None:
+        if isinstance(auto_create_sub_networks, str):
+            auto_create_sub_networks = auto_create_sub_networks.lower() == "true"
+        config["autoCreateSubnetworks"] = auto_create_sub_networks
+    else:
+        config["autoCreateSubnetworks"] = True  # default to subnet-mode network
+
+    # Optional routing config
+    if args.get("routing_config_routing_mode"):
+        config["routingConfig"] = {"routingMode": args["routing_config_routing_mode"]}
+
+    # Project ID
+    project = args.get("project_id")
+    # Build the compute service
+    compute = GCPServices.COMPUTE.build(creds)
+
+    # Execute the insert network request
+    response = compute.networks().insert(project=project, body=config).execute()
+
+    # Prepare output
+    data_res = {
+        "status": response.get("status"),
+        "kind": response.get("kind"),
+        "name": response.get("name"),
+        "id": response.get("id"),
+        "progress": response.get("progress"),
+        "operationType": response.get("operationType"),
+    }
+
+    headers = ["status", "kind", "id", "progress", "operationType", "name"]
+    readable_output = tableToMarkdown(
+        "Google Cloud Compute Network Insert", data_res, headers=headers, removeNull=True, headerTransform=pascalToSpace
+    )
+
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix="GCP.Compute.Operations",
+        outputs_key_field="id",
+        outputs=response,
+        raw_response=response,
+    )
+
+
+def gcp_compute_networks_list(creds: Credentials, args: dict[str, Any]) -> CommandResults:
+    """
+    Retrieves the list of networks available to the specified project.
+    """
+    project = args.get("project_id")
+    limit = (arg_to_number(args.get("limit")) or 500) if args.get("limit", "500") != "0" else 0
```

Comment:
```suggestion
    limit = (arg_to_number(args.get("limit")) or 500) if args.get("limit", "50") != "0" else 0
```
Let's change the default value to 50. Update it in the yml and README as well.